README.md

```diff
@@ -1,6 +1,15 @@
 <div align="center">
-    <img src="./logo/workflow.png" width="500" alt="Workflow Logo">
+    <img src="./logo/workflow.png" width="500" alt="Workflow Logo">    
+    <div  align="center">
+        <a style="padding: 0 10px" href="https://goreportcard.com/report/github.com/luno/workflow" > 
+            <img src="https://goreportcard.com/badge/github.com/luno/workflow"/> 
+        </a>
+        <a style="padding: 0 10px" href="https://codecov.io/gh/luno/workflow" > 
+            <img src="https://codecov.io/gh/luno/workflow/graph/badge.svg?token=YLMMOMHYOB"/> 
```

Comment:
Presume this token isn't sensitive? In any case, my preference would be to upload this to a sonar_report, which we can then show a badge for, e.g. https://github.com/getyoti/yoti-go-sdk/blob/master/.github/workflows/sonar.yaml which shows a badge on https://github.com/getyoti/yoti-go-sdk/tree/master

Reply:
No that token is public. Cool will go with sonar so that it helps track coverage.

Reply:
Yep it would be a lot better 😄

Reply:
Then this coverage metric won't just always say 0%! 

![image](https://github.com/user-attachments/assets/01c08a35-1cdf-4be9-ada7-9cbf2ac6ddde)